tools/Makefile

```diff
@@ -64,8 +67,8 @@ tools-y += sstrip
 tools-y += zip
 tools-y += zlib
 tools-y += zstd
-tools-$(if $(CONFIG_BUILD_ALL_HOST_TOOLS),y) += liblzo
```

Comment:
@Spudz76 it seems the rebase did some magic and the removal of this cause problem with sdk compilation (or compilation problem by using lzo compression... Can you by chance check this? I'm confused on why this actually works (considering liblzo is a dependency but in theory falls in tools-n list)

Reply:
If anything else pulls in `lzop` then lzop pulls in liblzo.  Squashfs4 code only needs liblzo, initramfs uses lzop.